oak_dice/src/evidence.rs

```diff
@@ -178,6 +178,15 @@ pub struct ApplicationKeys {
     pub encryption_public_key_certificate: [u8; CERTIFICATE_SIZE],
 }
 
+impl ApplicationKeys {
+    pub fn claims(&self) -> Result<coset::cwt::ClaimsSet, String> {
+        let decoded =
+            crate::utils::cbor_encoded_bytes_to_vec(&self.encryption_public_key_certificate[..])?;
+        crate::cert::get_claims_set_from_certificate_bytes(&decoded)
```

Comment:
I guess it this returns some form of `anyhow::Error`, which we then turn into a `String`, only to turn back into an `anyhow::error` in the `try_from`, above.
Could we make this function return an anyhow::Error as well, and just map the String returned by `cbor_encoded_bytes_to_vec` to anyhow::Error with `anyhow!`?

Reply:
That function also returns a string. Everything in this crate does, as it is used in stage0 where we don't want to include anyhow

Reply:
Fair enough. The main reason why we didn't want to use anyhow in stage0 is that anyhow requires an allocator, which we managed to avoid having in stage0 for quite some time.

However, these days we do have a heap allocator in stage0, so we may want to revisit the error handling situation at some point. But that's not for this PR :)